release/schedule/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: Release Schedule
+---
+
+WPE WebKit follows a 6-month development cycle, which means that:
+
+- Two major releases are done every year, typically in March and October.
+- Between major releases, any number of minor releases containing bug fixes
+  may be done.
+- Development releases can be produced at any time, of which release
+  candidates are done on the weeks prior to the next major release.
+
```

Comment:
I would avoid passive voice unless there is no choice. It is not natural (somehow it comes to us when we write technical content). Perhaps something like:


WPE WebKit follows a 6-month development cycle, where:

- There are two major releases per year, typically in March and September.
- Within a major release there might be minor releases containing bug fixes.
- Development releases are the base for the next major releases, and they do not follow a fixed schedule in the release cycle.

Reply:
I like your wording better, thanks!

Reply:
Also, I am going to replace “two major releases” with “two feature releases”; to avoid confusion later on when the page tells about the major version number.